exercises/concept/intro-select/intro-select.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE all_data AS
+    -- Task 1. Select all records.
+    SELECT "TODO: Replace this SELECT statement as needed to accomplish the task"
+;
+
+CREATE TABLE location_and_temperature AS
+    -- Task 2. Select only location and temperature data.
+    SELECT "TODO: Replace this SELECT statement as needed to accomplish the task"
+;
+
+CREATE TABLE seattle AS
+    -- Task 3. Select all data for Seattle.
+    SELECT "TODO: Replace this SELECT statement as needed to accomplish the task"
+;
+
+CREATE TABLE limited_humidity AS
+    -- Task 4. Select all data where the humidity is between 60% and 70%.
```

Comment:
I have no preference on whether this is `BETWEEN 60 AND 70` but I think it ought to be less ambiguous. The docs/stubs say 60% which I could be interpreted both as `60.0` and `0.60`.

Reply:
Whichever way you prefer, so long as it's not ambiguous ;)

Reply:
Replaced humidity with wind speed.